pages/program-details/[pid].tsx

```diff
@@ -1,16 +1,21 @@
 import { useRouter } from "next/router";
 import React from "react";
-import { Text, Spinner, Center } from "@chakra-ui/react";
+import { Box } from "@chakra-ui/react";
 import { useSession } from "next-auth/client";
 import { ProgramInfo } from "@components/ProgramInfo";
 import useSWR from "swr";
 import CardInfoUtil from "utils/cardInfoUtil";
+import useMe from "@utils/useMe";
+import { Loading } from "@components/Loading";
+import Participants from "@utils/containers/Participants";
 
-export const ProgramDetails: React.FC = () => {
-    const [session, loading] = useSession();
+const ProgramDetails: React.FC = () => {
     const router = useRouter();
     const { pid } = router.query;
 
+    const { me, isLoading } = useMe();
+    const [session, loading] = useSession();
```

Comment:
Nit: makes the variable names less ambiguous

```suggestion
    const [session, isSessionLoading] = useSession();
```